### PR TITLE
Feature: waitlist

### PR DESCRIPTION
**Pull request by kearfy**
Backend is pending still, don't merge

### CODE REVIEW

File: app/pages/join-waitlist.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      </div>
+      <Container>
+        <div className={styles.form}>
+          <Image src={Logo} alt="Kards logo" />
+          <div className={styles.inputs}>
+            <InputField
+              value={name}
+              label={nameLabel}
+              setValue={setName}
+              placeholder="Name"
```
Comment (ghost) [inline app/pages/join-waitlist.tsx]: Maybe specify what "name" users are supposed to put in here? Is it a username they can claim in advance, or their full name for contact purposes?

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/pages/join-waitlist.tsx (hunk 1 above)
Comment (kearfy) [reply app/pages/join-waitlist.tsx]: So like: "Enter your full name"?
I'd say that's clear enough for them to understand that it's about their first + last name...

Comment (ghost) [reply app/pages/join-waitlist.tsx]: Not necessarily, but you could make it "Full name" if that is the case! Again, I was personally confused about whether I'd put in the username I wanted or the name I can be addressed by in contact.

**Comment by kearfy** [comment]
Ready to merge, awaiting review

**Review by ghost — APPROVED** [review]

**Pull request merged**